src/Entity/IdeasWorkshop/Theme.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityNameSlugTrait;
+use AppBundle\Entity\ImageTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *     name="ideas_workshop_theme",
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="theme_slug_unique", columns="slug")
+ *     }
+ * )
+ *
+ * @UniqueEntity("slug")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Theme
+{
+    use EntityNameSlugTrait;
```

Comment:
the same question for the need of slug column for Theme entity?

Reply:
same answer, it will be exposed